spec/controllers/announcements_controller_spec.rb

```diff
@@ -130,4 +145,43 @@
       it_behaves_like 'access denied'
     end
   end
+  context 'check hide' do
```

Comment:
The string for this block should be the HTTP request, so `GET hide`. The truth is, if we were strictly following some kind of REST standard we would probably be using `POST` but I think we can just leave it for now.